packages/sommelier.rb

```diff
@@ -48,14 +48,22 @@ def self.install
   def self.postinstall
     puts
     puts "To complete the installation, execute the following:".lightblue
-    puts "echo '# Sommelier environment variables' >> ~/.bashrc".lightblue
+    puts "echo '# Sommelier environment variables + daemon' >> ~/.bashrc".lightblue
     puts "echo '# See https://github.com/dnschneid/crouton/wiki/Sommelier-(A-more-native-alternative-to-xiwi)' >> ~/.bashrc".lightblue
     puts "echo 'export GDK_BACKEND=wayland' >> ~/.bashrc".lightblue
     puts "echo 'export CLUTTER_BACKEND=wayland' >> ~/.bashrc".lightblue
     puts "echo 'export XDG_RUNTIME_DIR=/var/run/chrome' >> ~/.bashrc".lightblue
     puts "echo 'export WAYLAND_DISPLAY=wayland-0' >> ~/.bashrc".lightblue
     puts "echo 'export DISPLAY=:0' >> ~/.bashrc".lightblue
-    puts "echo '[ -f #{CREW_PREFIX}/sbin/sommelierd ] && #{CREW_PREFIX}/sbin/sommelierd &' >> ~/.bashrc".lightblue
+    puts "echo 'if [ ! -d /tmp/.X11-unix ]; then' >> ~/.bashrc".lightblue
+    puts "echo 'mkdir /tmp/.X11-unix' >> ~/.bashrc".lightblue
+    puts "echo 'fi' >> ~/.bashrc".lightblue
+    puts "echo 'chmod -R 1777 /tmp/.X11-unix' >> ~/.bashrc".lightblue
+    puts "echo 'sudo chown root:root /tmp/.X11-unix' >> ~/.bashrc".lightblue
+    puts "echo 'if ! pgrep -x \"sommelier\" > /dev/null' >> ~/.bashrc".lightblue
+    puts "echo 'then' >> ~/.bashrc".lightblue
+    puts "echo '/usr/local/sbin/sommelierd &' >> ~/.bashrc".lightblue
```

Comment:
Please change `/usr/local` to `#{CREW_PREFIX}`.

Reply:
K